### DISCUSSION TITLE
can we have api for merging the contacts?

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#5137] [Self hosted] [answered]
can we have api for merging the contacts?

**Comment by sojan-official** [comment, marked as answer]
ref: https://github.com/chatwoot/chatwoot/issues/5115#issuecomment-1195564326

```
POST:   api/v1/accounts/1/actions/contact_merge
PARAMS: { 
                    base_contact_id: 3
                    mergee_contact_id: 1
                }

```

- base contact should be the id of the contact you created/found via API 
- mergee contact id can be the contact with the random name ..

**Reply by NitinNR** [reply]
Can't we pass multiple ids in mergee_contact_id?

**Reply by sojan-official** [reply]
Nope, you will need to make individual requests for each contact at the moment

**Reply by NitinNR** [reply]
Do we have any plan to do so ?

**Reply by sojan-official** [reply]
@NitinNR it is not a priority at the moment. Since we have a workaround for this, at one can perform individual requests for each id
